ui/pages/swaps/import-token/import-token.js

```diff
@@ -47,7 +47,9 @@ export default function ImportToken({
       footer={ImportTokenModalFooter}
     >
       <Box
-        padding={[0, 6, 4, 6]}
+        paddingRight={6}
+        paddingBottom={4}
+        paddingLeft={4}
         alignItems={ALIGN_ITEMS.CENTER}
         display={DISPLAY.FLEX}
         className="import-token"
```

Comment:
Updating previous array props for different directions to singular direction props now that array props are reserved for responsive props